github-webhook.js

```diff
@@ -227,10 +231,18 @@ function handleRules (logStream, rules, event) {
       }
     })
 
+    cp.stdout.pipe(past)
```

Comment:
If `past` isn't piped anywhere else, i.e. there's no `logStream` or `rule.report` then it's just going to build up backpressure and eventually block the original stream. If there's enough data coming out of stdout and depending on how that impacts the child process, it could end up blocking it.
So it shouldn't be piped unless we know it's going to be piped to a downstream sink. So maybe conditional this on `if (logStream || rule.report)`